exotel.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/parnurzeal/gorequest"
+	"time"
+)
+
+//ExotelBaseURL is the base url for exotel api
+const ExotelBaseURL = "https://twilix.exotel.in"
+//ExotelRetryInterval is the retry interval for failed requests
+const ExotelRetryInterval = 3 //in Seconds
+
+//Exotel is the exte Exotel
+type Exotel struct {
+	AccountSid string
+	Token      string
+	requester  *gorequest.SuperAgent
+}
+
+//NewExotel creates a new
+func NewExotel(sid ,token string) *Exotel {
+	Exotel := &Exotel{requester: gorequest.New().SetBasicAuth(sid, token), AccountSid: sid, Token: token}
+	return Exotel
+}
+
+//SetHeaders Sets needed headers
+func (c *Exotel) setHeaders() *Exotel {
+	c.requester.
+		Set("User-Agent", "sheetblast-v1").
+		Set("Accept-language", "es").
+		Set("Content-Type", "application/x-www-form-urlencoded").
+		Set("accept", "application/json")
+	return c
+}
+
+//sendSMS sends sms  ,obviously  
+//This API is rate limited to 200 SMSes per minute. Once this limit has been crossed, your requests will be rejected with an HTTP 429 'Too Many Requests' code.
+func (c *Exotel) sendSMS(from string, to []string,body string) ( err error) {
```

Comment:
I think you can just merge `Send` and `sendSMS`, there's no reason to keep them separated.